src/ctelnet.h

```diff
@@ -41,6 +41,18 @@
 #include <queue>
 #include <string>
 
+#if defined (Q_OS_MACOS) || defined(Q_OS_LINUX)
```

Comment:
Gimme 24 hours and I'll report what the includes for FreeBSD need to be...

Reply:
Strange - I did not need ***any*** of these includes to get the code to compile (and you know how fussy my FreeBSD Clang compiler has been about other `#include`s recently) - do you definitely need these for Linux? {Given macOs's connection to \*BSDs unices they *may* not be needed for that one either}...

Reply:
If you do need to leave these in - can you put a comment in saying something about "FreeBSD does not seem to need them" (for anyone else who goes down the weird path I'm following!) :stuck_out_tongue_winking_eye:

Reply:
Right - FreeBSD **does** use all of the ones you have for the non-Windows case - but it also wants `#include <sys/types.h>` which is only a *recommendation* for Linux *I think*. It is possible that macOs will want that as well - see my comment outside of this review one which links macOs and \*BSDs...